examples/binary_payload/serve.py

```diff
@@ -23,7 +23,7 @@ def add_alpha(img):
 
 
 #  This is the overlay we will put on top of provided image
-OVERLAY = imread('ebaklya.png')
+OVERLAY = imread('examples/binary_payload/ebaklya.png')
```

Comment:
i think this examples are supposed to run from their respective directories. maybe at least make this path relative to `__file__`?

Reply:
Running examples from their directories is difficult. When running we should specify PYTHONPATH for Ebonite sources. If we are in example directory this leads to `../../src`-like things, moreover we should move to each directory if we want to consequtively run several examples.

Reply:
i thought that you run examples with legitimately installed ebonite (if you are not a developer). If you are a developer, this is resolved by IDE settings

Reply:
ok fixed